library/objective-c/EnvoyBridgeUtility.h

```diff
@@ -3,12 +3,26 @@
 #import "library/common/types/c_types.h"
 
 static inline envoy_data toNativeData(NSData *data) {
+  if (data == nil) {
+    return envoy_nodata;
+  }
+
   uint8_t *native_bytes = (uint8_t *)safe_malloc(sizeof(uint8_t) * data.length);
   memcpy(native_bytes, data.bytes, data.length);
   envoy_data ret = {data.length, native_bytes, free, native_bytes};
   return ret;
 }
 
+static inline envoy_data *toNativeDataPtr(NSData *data) {
+  if (data == nil) {
+    return nil;
```

Comment:
Below you're using `NULL`, can we keep these consistent?

Reply:
Yeah, this should be `NULL` because it's not a valid Objective-C object.